test/blocks/dc-converter-widget/mocks/body_gen_cache.html

```diff
@@ -0,0 +1,16 @@
+<main>
+    <div class="dc-converter-widget">
+        <div>
+            <div>pdf-to-ppt</div>
+        </div>
+        <div>
+            <div>https://www.adobe.com/go/testredirect</div>
+        </div>    
+        <div>
+            <div>https://www.adobe.com/generate_cache_url</div>
```

Comment:
are we even using this?

Reply:
@peller We can revisit the design. If the feature is not used and will never be used, we should remove the related code.